### PR TITLE
[zh-cn] translate device memory api

### PR DESCRIPTION
**Pull request by serenaczx**
### Description

[zh-CN] translate device memory api

### CODE REVIEW

File: files/zh-cn/web/api/device_memory_api/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- {{HTTPHeader("Device-Memory")}} header
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/device_memory_api/index.md:47]:
```suggestion
- {{HTTPHeader("Device-Memory")}} 标头
```

File: files/zh-cn/web/api/device_memory_api/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: 设备内存 API
+slug: Web/API/Device_Memory_API
+l10n:
+  sourceCommit: ca577adc00ddc882765c131739ad2ed25edd2285
+---
+
+{{DefaultAPISidebar("Device Memory API")}}{{securecontext_header}}{{AvailableInWorkers}}
+
+客户端设备的性能在很大程度上取决于可用内存（RAM）的大小。传统上，开发人员必须使用启发式方法，要么对设备进行基准测试，要么根据设备制造商或 User Agent 字符串等其他参数推断设备的性能。
```
Comment (yin1999) [inline files/zh-cn/web/api/device_memory_api/index.md:10]:
```suggestion
客户端设备的性能在很大程度上取决于可用内存（RAM）的大小。传统上，开发人员必须使用启发式方法，要么对设备进行基准测试，要么根据设备制造商或用户代理字符串等其他参数推断设备的性能。
```

File: files/zh-cn/web/api/device_memory_api/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## 获取设备内存
```
Comment (yin1999) [inline files/zh-cn/web/api/device_memory_api/index.md:12]:
```suggestion
## 确定设备内存
```

File: files/zh-cn/web/api/device_memory_api/index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+有两种方法可以获取设备的 RAM 大小：使用设备内存 JavaScript API 或接受 Client Hints
```
Comment (yin1999) [inline files/zh-cn/web/api/device_memory_api/index.md:14]:
```suggestion
有两种方法可以确定设备的 RAM 大小：使用设备内存 JavaScript API 或接受客户端提示。
```

File: files/zh-cn/web/api/device_memory_api/index.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+### JavaScript API
+
+你可以通过检索 {{DOMxRef("Navigator.deviceMemory")}} 或 {{DOMxRef("WorkerNavigator.deviceMemory")}} 来查询设备的近似 RAM 的大小。
+
+```js
+const RAM = navigator.deviceMemory;
+```
+
+### Client Hints
```
Comment (yin1999) [inline files/zh-cn/web/api/device_memory_api/index.md:24]:
```suggestion
### 客户端提示
```

File: files/zh-cn/web/api/device_memory_api/index.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+你也可以使用 [Client Hints](/zh-CN/docs/Web/HTTP/Client_hints) HTTP 标头与 `Device-Memory` 指令来检索设备的近似 RAM 大小。
```
Comment (yin1999) [inline files/zh-cn/web/api/device_memory_api/index.md:26]:
```suggestion
你也可以使用[客户端提示](/zh-CN/docs/Web/HTTP/Client_hints) HTTP 标头与 `Device-Memory` 指令来检索设备的近似 RAM 大小。
```

File: files/zh-cn/web/api/device_memory_api/index.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+## 接口
+
+### 对其他接口的扩展
+
+- {{domxref("Navigator.deviceMemory")}} {{ReadOnlyInline}}
+  - : 返回以 GB 为单位的设备内存的大致数量。
+- {{domxref("WorkerNavigator.deviceMemory")}} {{ReadOnlyInline}}
+  - : 返回以 GB 为单位的设备内存的大致数量。
```
Comment (yin1999) [inline files/zh-cn/web/api/device_memory_api/index.md:35]:
```suggestion
  - : 返回以 GB 为单位的设备内存的近似大小。
- {{domxref("WorkerNavigator.deviceMemory")}} {{ReadOnlyInline}}
  - : 返回以 GB 为单位的设备内存的近似大小。
```

**Review by yin1999 — APPROVED** [review]
Thank you @serenaczx. I've pushed a commit to refine the translation. Welcome to the community :tada:

**Pull request merged**